graphics/vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,62 @@
+"""
+render 3d points for 2d surfaces.
+"""
+
+import math
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+
+def convert_to_2d(x: float, y: float, z: float, scale: float,
+                  distance: float) -> (float, float):
+    """
+    Converts 3d point to a 2d drawable point
+
+    >>> convert_to_2d(1.0, 2.0, 3.0, 10.0, 10.0)
+    (7.6923076923076925, 15.384615384615385)
+    """
+    if not all(isinstance(val, (float, int)) for val in locals().values()):
+        raise TypeError("Input values must either be float or int: "
```

Comment:
This is correct.  I was wrong about this and @dhruvmanila was right.  It is good to learn new things.  Thanks

Reply:
Hey, @cclauss anytime! 😁 
But I don't understand what were you wrong about or what am I right about?

Reply:
Hello, @cclauss told me to use `ValueError` instead of `TypeError` so he meant to say that @dhruvmanila is right